src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -572,36 +572,57 @@ def record_metadata_proportions(final_output_dir: Path) -> None:
             dataset_metadata.groupby(groupby_cols)[aggregate_count_cols].sum().reset_index()
         )
 
+        # Denominator columns
+        denominator_columns_mapper = {
+            "row_noise.row_probability_in_households_under_18": "group_rows.row_probability_in_households_under_18",
+            "row_noise.row_probability_in_college_group_quarters_under_24": "group_rows.row_probability_in_college_group_quarters_under_24",
+        }
+
+        denominator_columns = (
+            list(denominator_columns_mapper.values())
+            + [data_keys.METADATA_COLUMNS.NUMBER_OF_ROWS]
+            if set(denominator_columns_mapper.values()).issubset(dataset_metadata.columns)
+            else [data_keys.METADATA_COLUMNS.NUMBER_OF_ROWS]
+        )
```

Comment:
Are you doing this for any reason other than that you are getting an error when you try to drop columns that don't exist? If so, I'd suggest you do this ...
```suggestion
        denominator_columns = list(denominator_columns_mapper.values()) + [data_keys.METADATA_COLUMNS.NUMBER_OF_ROWS]
```